### ISSUE TITLE
OPCUA plugin to receive data from OPCUA server

### ISSUE THREAD

**Issue by krgrsebastian** [redpanda-data/connect#1651]
### Description:
I am working on a project that requires data from an OPCUA server to be forwarded into Benthos. An OPCUA plugin for Benthos would be very helpful in achieving this. 

As OPCUA is still relevant in the industrial internet of things (IIoT), a plugin for Benthos would be useful for many users. OPCUA is a communication protocol designed to facilitate the interoperability of devices and systems from different vendors and to enable secure and reliable data exchange in real-time. It is widely adopted by industrial automation vendors and is suitable for use in complex and large-scale IIoT environments.

### Steps to reproduce:
N/A

### Expected behavior:
A plugin that allows Benthos to receive data from an OPCUA server.

Actual behavior:
There is currently no OPCUA plugin available for Benthos.

Helpful library: https://github.com/gopcua/opcua

**Comment by ivenhowe** [comment]
The United Manufacturing Hub maintains a suitable project based on benthos: https://github.com/united-manufacturing-hub/benthos-umh. However, it would be nice to add support for IoT protocols in bathos natively.

**Comment by JeremyTheocharis** [comment]
We are already talking with @Jeffail about merging it, but it would only make sense once the plugin is finished (read, write, etc.) and we have a plan on maintaining it.

**Comment by twmb** [comment]
An external OPC-UA plugin exists in the United Manufacturing Hub project (benthos-umh). The discussed merge into core never materialized. Closing as stale — the external plugin approach is the right path for niche protocols. If there's renewed interest in contributing this upstream, feel free to open a new issue or PR.
